frame/3/bli_l3_ind.c

```diff
@@ -58,7 +58,7 @@ bool bli_l3_ind_oper_st[BLIS_NUM_IND_METHODS][BLIS_NUM_LEVEL3_OPS][2] =
         /*   gemm           gemmt          hemm           herk           her2k          symm
              syrk           syr2k          trmm3          trmm           trsm  */
         /*    c     z    */
-/* 1m   */ { {FALSE,FALSE}, {FALSE,FALSE}, {FALSE,FALSE}, {FALSE,FALSE}, {FALSE,FALSE}, {FALSE,FALSE},
+/* 1m   */ { {TRUE,TRUE}, {FALSE,FALSE}, {FALSE,FALSE}, {FALSE,FALSE}, {FALSE,FALSE}, {FALSE,FALSE},
```

Comment:
@BhaskarNallani It appears that you are enabling 1m by default for `gemm`, which will cause it to have precedence over a native microkernel (if it exists). Can you elaborate on why this change was made / needed?

If your goal is for `gemm3m` to use the 1m implementation of `gemm`, I think we will need to find another way to make this happen.

Reply:
FYI: I have found a more elegant solution to this, which I am in the process of implementing.